src/components/org/OrgAdmin/index.tsx

```diff
@@ -272,11 +275,79 @@ function OrgAdmin() {
         ],
       })),
     };
+
     sendMutate(requestBody);
   };
 
+  const handleClickMagicButton = (type: 'SET' | 'GET' | 'IMAGE' | 'RESET') => {
+    if (type === 'SET') {
+      localStorage.setItem('org-admin', JSON.stringify(getValues()));
+      alert('데이터 임시저장 성공!');
+      return;
+    }
+    if (type === 'GET') {
+      const data = localStorage.getItem('org-admin');
+      if (!data) {
+        alert('저장된 데이터가 없음ㅠ');
+      } else {
+        reset(JSON.parse(data));
+        alert(
+          '데이터 불러오기 성공! \n소개탭 헤더를 반드시 다시 첨부해주세요!',
+        );
+      }
+      return;
+    }
+
+    if (type === 'IMAGE') {
+      const headerImage = getValues('headerImageFileName');
+      if (!(headerImage.file instanceof File)) {
```

Comment:
```suggestion
      if (!(headerImage?.file instanceof File)) {
```
? 없어서 이미지 채우기 눌렀을 때 에러 떠요

Reply:
감사합니다!

Reply:
확인해주시면 머지할게여!

Reply:
확인했습니다! 고생하셨어여 ◡̈